### PR TITLE
Add camel-bonita wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-bonita wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -548,6 +548,25 @@
         <bundle dependency='true'>mvn:com.ibm.icu/icu4j/${icu4j-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-bindy/${project.version}</bundle>
     </feature>
+    <feature name='camel-bonita' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-blueprint</feature>
+        <feature>jackson</feature>
+        <feature>jakarta-jws</feature>
+        <feature version="[11,12)">jetty</feature>
+        <bundle dependency="true">mvn:jakarta.servlet/jakarta.servlet-api/5.0.0</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: I would add a feature for this version of the servlet api

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -548,6 +548,25 @@
...
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.wsdl4j/1.6.3_1</bundle>
+        <bundle dependency="true">mvn:org.apache.ws.xmlschema/xmlschema-core/2.3.1</bundle>
+        <bundle dependency="true">mvn:org.apache.neethi/neethi/3.2.0</bundle>
+        <bundle dependency="true">mvn:org.glassfish.hk2/osgi-resource-locator/2.5.0-b42</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-base/${jackson2-version}</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-json-provider/${jackson2-version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.cxf/cxf-rt-frontend-jaxrs/${cxf-version}</bundle>
+        <bundle>mvn:org.apache.camel.karaf/camel-attachments/${project.version}</bundle>
+        <bundle>mvn:org.apache.camel.karaf/camel-http-base/${project.version}</bundle>
+        <bundle>mvn:org.apache.camel.karaf/camel-cxf-all/${project.version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: I would avoid depending on `camel-cxf-all` as it brings a lot of dependencies that are useless for bonita such as `camel-blueprint` and many others. Instead add pure cxf dependencies

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: ok done

**Review by essobedo — APPROVED** [review]
Awesome 👍

**Pull request merged**